IFC4x3/Properties/s/SignalWalkwayLength_2hoc6PPhvELAtWTUyKoQU6/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicates the length of the walkway from signal to signal post telephone.
```

Comment:
Not sure I understand this. Should it be "Indicates the length of the walkway from signal to signal telephone post ."?